projects/frontend/shared-components/gui/angular.json

```diff
@@ -0,0 +1,214 @@
+            "codeCoverage": true,
+            "codeCoverageExclude": [
+              "projects/shared/src/**/*component.ts",
+              "projects/shared/src/lib/unit-testing/**/*"
+            ],
+            "sourceMap": {
+              "scripts": true,
+              "vendor": true,
+              "styles": true
+            }
+          }
+        },
+        "lint": {
+          "builder": "@angular-eslint/builder:lint",
+          "options": {
+            "lintFilePatterns": [
+              "projects/shared/**/*.ts",
+              "projects/shared/**/*.html"
+            ]
+          }
+        }
+      },
+      "schematics": {
+        "@schematics/angular:component": {
+          "style": "scss"
+        }
+      }
+    },
+    "documentation-ui": {
+      "projectType": "application",
+      "root": "projects/documentation-ui",
+      "sourceRoot": "projects/documentation-ui/src",
+      "prefix": "app",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist/documentation-ui",
+            "index": "projects/documentation-ui/src/index.html",
+            "main": "projects/documentation-ui/src/main.ts",
+            "polyfills": "projects/documentation-ui/src/polyfills.ts",
+            "tsConfig": "projects/documentation-ui/tsconfig.app.json",
+            "allowedCommonJsDependencies": [
+              "dagre",
+              "webcola"
+            ],
+            "assets": [
+              "projects/documentation-ui/src/favicon.ico",
+              "projects/documentation-ui/src/assets",
+              {
+                "glob": "clr-ui*.min.*",
+                "input": "node_modules/@clr/ui",
+                "output": "assets/css"
+              },
+              {
+                "glob": "**/*",
+                "input": "node_modules/@vdk/shared/assets",
+                "output": "/assets/"
+              }
+            ],
+            "styles": [
+              "node_modules/@clr/icons/clr-icons.min.css",
+              "projects/documentation-ui/src/styles.scss"
+            ],
+            "scripts": [
+              "node_modules/@clr/icons/clr-icons.min.js"
+            ]
+          },
+          "configurations": {
+            "development": {
+              "sourceMap": {
+                "scripts": true,
+                "vendor": true,
+                "styles": true
+              },
+              "vendorChunk": true,
+              "extractLicenses": false,
+              "buildOptimizer": false,
+              "optimization": false,
+              "namedChunks": true,
+              "preserveSymlinks": true
+            },
+            "production": {
+              "fileReplacements": [
+                {
+                  "replace": "projects/documentation-ui/src/environments/environment.ts",
+                  "with": "projects/documentation-ui/src/environments/environment.prod.ts"
```

Comment:
So we have basically empty documentation-ui screens at this point.
Should we keep those (`documentation-ui/`) - in case we would in the future need to visualise some newly added components, or should we simply not add those until we need to actually visualise something?

In case we decide to keep it, we need to document the environments supported by `documentation-ui`

Reply:
I don't have much context, maybe @tozka will have some valuable input. That being said, I'm generally against introducing code that "we might need some day". It clutters the codebase.

Reply:
I fancy the simple approach of only including what's needed, so in favour of dropping `documentation-ui/` from here + VEP 1507(in additional PR)

Reply:
https://github.com/vmware/versatile-data-kit/issues/1644